src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -248,6 +248,21 @@ func (r *Cluster) validatePandaproxyListeners() field.ErrorList {
 		}
 	}
 
+	// for now only one listener can have TLS to be backward compatible with v1alpha1 API
```

Comment:
I think we should also validate that kafka API tls is also enabled and has client auth enabled if proxy has these enabled, right?

Reply:
We generate separate certs for pandaproxy (and TLS settings). Perhaps we could share them for simplicity. We can think about it but I think it's good for now.

Reply:
oh okay, then you need to update the comment here https://github.com/vectorizedio/redpanda/pull/1339/files#diff-08b68e2edc65d6141e69e9202b8e692362ec843626fa4b64778bb3acfaf03194R276